Bug 2056469, Bug 2056470 - fix connect-existing docs and throw explicit error for sessions without BiDi (#126)
freema/firefox-devtools-mcp: README.md

### Connect to existing Firefox

Use `--connect-existing` to automate your real browsing session, with cookies, logins, and open tabs intact:

```bash
# Start Firefox with Marionette and the Remote Agent (BiDi)
firefox --marionette --remote-debugging-port

# Run the MCP server
npx @mozilla/firefox-devtools-mcp --connect-existing --marionette-port 2828
```

> Both flags are required. `--marionette` lets the MCP attach to the running Firefox, and `--remote-debugging-port` starts the Remote Agent that provides the WebDriver BiDi endpoint used by navigation, console, and network tools. Setting `marionette.enabled` in `about:config` or `user.js` is not sufficient on its own, because the Remote Agent only starts when the command line flag is passed. If Firefox is running without it, the MCP server fails to connect and asks you to restart Firefox with both flags.

Reviewer: Hmmm I think the marionette.enabled preference was a complete hallucination, which I missed to remove. This preference doesn't exist as far as I know, let's drop the whole section about it. And again, no need to explain which tools current use webdriver BiDi.
```suggestion
Both flags are required because the MCP uses both WebDriver Classic (`--marionette`) and WebDriver BiDi (`--remote-debugging-port`). If Firefox is only started with `--marionette`, the MCP server fails to connect and asks you to restart Firefox with both flags.
```
